<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderListViewController</class>
 <widget class="QWidget" name="OrderListViewController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>651</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="Sort_user">
      <item>
       <property name="text">
        <string>全部</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>待付款</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>待收货</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>待评论</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>已完成</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>已取消</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="Sort_merUpper">
      <item>
       <property name="text">
        <string>未完成</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>价格</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>评分</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>179</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="OrderCode">
      <property name="placeholderText">
       <string>输入订单号搜索</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Search">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>651</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="Sort_Time">
      <item>
       <property name="text">
        <string>近三天</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>近一周</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>本月</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>本季</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>近一年</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="StartTimeSymbol">
      <property name="text">
       <string>开始时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="StartTime">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="EndTimeSymbol">
      <property name="text">
       <string>结束时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="EndTime">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
